src/renderer/components/editor.tsx

```diff
@@ -75,14 +75,18 @@ export class Editor extends React.Component<EditorProps> {
    * Initialize Monaco.
    */
   public async initMonaco() {
-    const { monaco, monacoOptions: monacoOptions } = this.props;
+    const { monaco, monacoOptions: monacoOptions, appState } = this.props;
     const ref = this.containerRef.current;
 
+    const { fontFamily, fontSize } = appState;
+
     if (ref) {
       this.editor = monaco.editor.create(ref, {
         automaticLayout: true,
         language: this.language,
         theme: 'main',
+        ...(fontFamily && { fontFamily }),
+        ...(fontSize && { fontSize }),
```

Comment:
```suggestion
        fontFamily,
        fontSize,
```

Reply:
What motivates this suggestion? Both of these can be undefined, and therefore we wouldn't want to pass these values as `undefined` - we only want to set them when they're set, otherwise we would leave them as default.

Reply:
e.g:
<img width="562" alt="image" src="https://user-images.githubusercontent.com/8198408/167617595-51506b20-2eb4-4598-9d4f-48c4a691bd9c.png">

And I've just actually tested it.
When the value is undefined, the result is the same as if `fontFamily` and `fontSize` were not passed in.
In fact, it is a means of optimizing the code.